tests/test_marshmallow_loader.py

```diff
@@ -135,6 +135,29 @@ def test_marshmallow_load_nested_errors(app, db, es, test_data, search_url,
             response_json['errors'][0]['field'] in ('nested_field', 'nested')
 
 
+def test_marshmallow_load_nested_subfield_errors(app, db, es, test_data,
+                                                 search_url, search_class):
+    """Test loading nested subfield errors."""
+    app.config['RECORDS_REST_DEFAULT_LOADERS'] = {
+        'application/json': marshmallow_loader(_TestSchemaNested)}
+
+    with app.test_client() as client:
+        HEADERS = [
+            ('Accept', 'application/json'),
+            ('Content-Type', 'application/json')
+        ]
+
+        # Create record
+        missing_top_most_required_field = dict(nested_field=[{}])
+        res = client.post(
+            search_url, data=json.dumps(missing_top_most_required_field),
+            headers=HEADERS)
+        assert res.status_code == 400
+        response_json = json.loads(res.data.decode('utf-8'))
+        assert response_json['errors'][0]['field'] == 'sample'
+        assert response_json['errors'][0]['parents'] == ['nested_field', 0]
```

Comment:
For completeness, I would also add a test regarding a nested non-list field, i.e. to see that 'parents': ['non_list_nested_field', 'sample']` works as expected.